mappings/net/minecraft/item/ItemStack.mapping

```diff
@@ -194,10 +228,18 @@ CLASS net/minecraft/class_1799 net/minecraft/item/ItemStack
 		ARG 3 breakCallback
 	METHOD method_7957 updateEmptyState ()V
 	METHOD method_7958 hasGlint ()Z
-	METHOD method_7959 putSubTag (Ljava/lang/String;Lnet/minecraft/class_2520;)V
+	METHOD method_7959 putNbtElement (Ljava/lang/String;Lnet/minecraft/class_2520;)V
```

Comment:
(this has already been mentioned) `Element` hasn't been used anywhere else - why here?

Reply:
Because the argument taken here is a NbtElement, while most of the other methods take a NbtCompound in.

Reply:
Hmm, OK. If there's a technical reason for it being different, then I'm happy to approve it if others do.